src/DropdownButton.js

```diff
@@ -1,57 +1,57 @@
-import React from 'react';
-import Dropdown from './Dropdown';
 import omit from 'lodash-compat/object/omit';
 import pick from 'lodash-compat/object/pick';
-import Button from './Button';
+import React from 'react';
 
-class DropdownButton extends React.Component {
+import Dropdown from './Dropdown';
+import { bsClass, prefix } from './utils/bootstrapUtils';
+
+const propTypes = {
+  ...Dropdown.propTypes,
+
+  // Toggle props.
+  bsStyle: React.PropTypes.string,
+  bsSize: React.PropTypes.string,
+  title: React.PropTypes.node.isRequired,
+  noCaret: React.PropTypes.bool,
+
+  // Override generated docs from <Dropdown>.
+  children: React.PropTypes.node,
+};
 
+class DropdownButton extends React.Component {
   render() {
-    let { bsStyle, bsSize, disabled } = this.props;
-    let { title, children, ...props } = this.props;
+    const { bsSize, bsStyle, title, children, ...props } = this.props;
 
-    let dropdownProps = pick(props, Object.keys(Dropdown.ControlledComponent.propTypes));
-    let toggleProps = omit(props, Object.keys(Dropdown.ControlledComponent.propTypes));
+    const dropdownProps = pick(
+      props, Object.keys(Dropdown.ControlledComponent.propTypes)
+    );
+    const toggleProps = omit(
+      props, Object.keys(Dropdown.ControlledComponent.propTypes)
+    );
 
     return (
-      <Dropdown {...dropdownProps}
+      <Dropdown
+        {...dropdownProps}
         bsSize={bsSize}
         bsStyle={bsStyle}
       >
         <Dropdown.Toggle
           {...toggleProps}
-          disabled={disabled}
+          bsClass={prefix(props, 'toggle')}
+          bsSize={bsSize}
+          bsStyle={bsStyle}
         >
           {title}
         </Dropdown.Toggle>
-        <Dropdown.Menu>
+
+        <Dropdown.Menu bsClass={prefix(props, 'menu')}>
```

Comment:
It also effectively makes setting bsClass on the subcomponents (or changing the defaults) a noop though right? Since Dropdown will always have a default for this.